README.md

```diff
@@ -8,11 +8,11 @@ available in your resources.
 Add to your project.clj's dependencies:
 
 ```clojure
-[turbovote.resource-config "0.2.1"]
+[democracyworks/resource-config "0.2.2-SNAPSHOT"]
```

Comment:
Maybe replace this with a Clojars badge so we don't have to keep on it? It wouldn't work until the rename, of course.